paws.common/R/config.R

```diff
@@ -182,14 +182,53 @@ get_instance_metadata <- function(query_path = "") {
   if (trimws(tolower(get_env("AWS_EC2_METADATA_DISABLED"))) %in% c("true", "1")) {
     return(NULL)
   }
-
+  # Get token timeout for IMDSv2 tokens
+  tokentimeout=trimws(tolower(get_env("PAWS_EC2_IMDSV2_TOKEN_TIMEOUT_SECONDS")))
+  if (is.na(as.numeric(tokentimeout))) {
+        tokentimeout="30"
```

Comment:
Shouldn't we try and mimic what is being done in botocore? and use  `'21600'` as  our token timeout? https://github.com/boto/botocore/blob/master/botocore/utils.py#L376